api/argo-templates/render.go

```diff
@@ -43,8 +43,9 @@ func GetWorkflowTemplate() []byte {
 	return argoWorkflowTemplate
 }
 
-func GetWrokerRebuildWorkflow(hostname string, xName string) ([]byte, error) {
-	err := validator.ValidateWorkerHostname(hostname)
+func GetWrokerRebuildWorkflow(hostnames []string, xName string) ([]byte, error) {
```

Comment:
Should this be GetWorker.. instead of GetWroker?

Reply:
yep, @mitchty caught this too. i will fix it when i start working on render function CASMPET-5466